packages/flutter_tools/lib/src/error_messages.dart

```diff
@@ -0,0 +1,177 @@
+import 'base/context.dart';
+import 'ios/cocoapods.dart';
+import 'vscode/vscode_validator.dart';
+
+ErrorMessages get errorMessages => context[ErrorMessages];
+
+/// Class containing the messages that can be produced by the doctor validators.
+/// Members are split by validator and are given in the order they appear in the
```

Comment:
This comment won't stay accurate for long. The only way message ordering makes sense is alphabetical. So let's either do that or not assume any ordering. It is OK that you split them by validator. Unless code reviewers are careful though that kind of separation does not last very long.

Reply:
Done. Removed last sentence.